hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/HoodieTable.java

```diff
@@ -157,6 +157,15 @@ private synchronized FileSystemViewManager getViewManager() {
    */
   public abstract HoodieWriteMetadata<O> delete(HoodieEngineContext context, String instantTime, K keys);
 
+  /**
+   * Deletes all data of partitions.
+   * @param context    HoodieEngineContext
+   * @param instantTime Instant Time for the action
+   * @param partitions   {@link List} of partition to be deleted
+   * @return HoodieWriteMetadata
+   */
+  public abstract HoodieWriteMetadata deletePartitions(HoodieEngineContext context, String instantTime, List<String> partitions);
```

Comment:
Do you think using JavaRDD<String> for partitions makes more sense (given all other APIs are using RDD)?

Reply:
thanks, i thinks "List<String>" will be ok. Because flink and spark will implement HoodieTable, use javaRDD not suitable to flink.